community/modules/scheduler/schedmd-slurm-gcp-v5-controller/outputs.tf

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "controller_instance_id" {
+  description = "Self link of the controller node"
+  value       = module.slurm_controller_instance.slurm_controller_instance.instances_details[0].id
+}
+
+output "controller_instance" {
+  description = "Self link of the controller node"
```

Comment:
The description is repeated

Reply:
Updated both description, thanks for catching that.